package/setup.py

```diff
@@ -353,6 +353,13 @@ def extensions(config):
                          include_dirs=include_dirs + ['MDAnalysis/lib/include'],
                          define_macros=define_macros,
                          extra_compile_args=cpp_extra_compile_args)
+    aug = MDAExtension('MDAnalysis.lib._augment',
+                         sources=['MDAnalysis/lib/_augment' + source_suffix],
+                         language='c++',
+                         libraries=mathlib,
```

Comment:
c++ doesn't need to link against a math library. This is part of the stblibc++ (or libc++ on osx). Any c++ codes links by default against this library.

Reply:
Yes it doesn't. Thanks for pointing it out.